src/main.cpp

```diff
@@ -7116,6 +7116,12 @@ bool static ProcessMessage(CNode* pfrom, string strCommand, CDataStream& vRecv,
             }
             else if (neural_request=="neural_hash")
             {
+            if(0==neural_request_id.compare(0,13,"supercfwd.rqa"))
```

Comment:
why can't this be compared directly? In miner.cpp:686 it looks like "supercfwd.rqa" is the entire content of the request id.

Reply:
This is for extensibility. I am re-using the request-ID field and there might be a need for a actual request ID.